### ISSUE TITLE
Problem in environment.yml

### ISSUE THREAD

**Issue by boweny920** [BackofenLab/Acr#4]
environment.yml Line 81 : - python-g 0 -dateutil==2.8.1 
Gave rise to the following error with conda: 
    **Pip subprocess output:
    
    Pip subprocess error:
    Usage: __main__.py [options]
    
    ERROR: Invalid requirement: python-g 0 -dateutil==2.8.1
    __main__.py: error: no such option: -d
    
    failed
    
    CondaEnvException: Pip failed**

I am guessing this is a formatting issue in file environment.yml, what are you trying to install with "- python-g 0 -dateutil==2.8.1" ? 

Thanks !

**Comment by k6logc** [comment]
Hello,
We are seeing the same issue, adding it to the yml did not resolve it.
Any updates on this issue?
@boweny920 were you able to resolve it?
Thank you!

**Comment by boweny920** [comment]
> Hello, We are seeing the same issue, adding it to the yml did not resolve it. Any updates on this issue? @boweny920 were you able to resolve it? Thank you!

No, i have not. Still waiting for the authors to address a potential solution. However, in the mean time there are other tools you can try, like AcrFinder, PaCRISPR. 

Hope this helps!

**Comment by snayfach** [comment]
I ended up blasting versus the Gussow et al. database which worked fairly well
http://acrcatalog.pythonanywhere.com/download/
